src/app/app.js

```diff
@@ -1,4 +1,6 @@
 // Special Debug Console Calls!
+var localStorage = window.localStorage || {};
```

Comment:
haha, ok, i'll push that change, that branch needs more work from what we did on the hackathon anyway.